packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -162,30 +172,30 @@ pub async fn graph_root_latest(
     #[cfg(feature = "metrics")]
     METRICS.db.postgres.graph_root_latest_calls.inc();
 
-    let row = sqlx::query(&format!(
-        r#"
-        SELECT *
-        FROM graph_registry_graph_root
-        WHERE schema_name = '{namespace}' AND schema_identifier = '{identifier}'
-        ORDER BY id DESC LIMIT 1"#
-    ))
+    let row = sqlx::query(
+        "SELECT * FROM graph_registry_graph_root
+        WHERE schema_name = $1 AND schema_identifier = $2
+        ORDER BY id DESC LIMIT 1",
+    )
+    .bind(namespace)
+    .bind(identifier)
     .fetch_one(conn)
     .await?;
 
     let id: i64 = row.get(0);
     let version: String = row.get(1);
     let schema_name: String = row.get(2);
-    let query: String = row.get(3);
-    let schema: String = row.get(4);
-    let schema_identifier: String = row.get(5);
+    let schema_identifier: String = row.get(3);
+    let query: String = row.get(4);
+    let schema: String = row.get(5);
```

Comment:
Is there a reason that the order was changed here? This is in the incorrect order and will break querying in the GraphQL API.

Reply:
I copied out the order in the struct 
```
#[derive(Debug)]
pub struct GraphRoot {
    pub id: i64,
    pub version: String,
    pub schema_name: String,
    pub schema_identifier: String,
    pub query: String,
    pub schema: String,
}
```
apart from referring to master, is there another way I should know what the correct order would be? Just wondering if there is some piece I'm not seeing here.

Reply:
I know these fields are not ordered, but it was an attempt. Albeit a fallible one!

Reply:
Unfortunately, you'd probably have to look at the migrations and determine the order as new columns would be added to the end. Alternatively, you could also do a `SELECT` against the table to see the structure. These `query_as!` macros did a lot of magic under the hood to match the columns and such, so we just have to make sure that we're checking the order.

Reply:
migrations as source of truth sounds good. just updating these now. Thanks!